src/Resources/contao/languages/de/tl_style_manager.xlf

```diff
@@ -230,6 +230,30 @@
         <source>Select the modules that this group may be displayed for.</source>
         <target>Wählen Sie die Module aus, für die diese Gruppe angezeigt werden soll.</target>
       </trans-unit>
+      <trans-unit id="tl_style_manager.formFieldsInverted.0">
+        <source>Excluded form fields</source>
+        <target>Ausgeschlossene Formularfelder</target>
+      </trans-unit>
+      <trans-unit id="tl_style_manager.formFieldsInverted.1">
+        <source>Select the form fields that this group may be &lt;strong&gt;not&lt;/strong&gt; displayed for (Inverted mode active).</source>
+        <target>Wählen Sie die Formularfelder aus, die für diese Gruppe &lt;strong&gt;nicht&lt;/strong&gt; angezeigt werden sollen (Invertierungsmodus aktiv).</target>
+      </trans-unit>
+      <trans-unit id="tl_style_manager.modulesInverted.0">
+        <source>Excluded modules</source>
+        <target>Ausgeschlossene Module</target>
```

Comment:
To make things more clear, I started using a <strong> tag rather than writing the "NOT" uppercase. 
I also changed the translation of the Label.